src/lib/svg_text_utils.js

```diff
@@ -91,7 +91,7 @@ exports.convertToTspans = function(_context, _callback) {
         }
         _context.text('')
             .style({
-                visibility: 'visible',
+                visibility: 'inherit',
```

Comment:
This is definitively the first step to take. :+1: 

My only concern here is possible incompatibilities in Illustrator and svg conversion tools (e.g. Batik).

To avoid any rolling :game_die: , I'm thinking about adding a few lines to the [`toSVG`](https://github.com/plotly/plotly.js/blob/708e0ffd9ee827fd2640334a072f86605dcb2f5d/src/snapshot/tosvg.js#L71) routine making sure all exported images get `visibility: 'visible'` as opposed to `inherit` while interactive graphs would get `'inherit'`.